test/README.md

```diff
@@ -52,14 +52,14 @@ pack the repo into thunk form
 I have found that the best way to hack on the tests involve some sort of environement such as the following
 ```
  cd nix/dep/flextesa-dev
- nix-shell -p ocamlPackages.utop opam ocamlPackages.dune m4 pkgconfig gmp hidapi                   
- opam init --root=.                                                                                
- eval $(opam env)                                                                                  
+ nix-shell -p ocamlPackages.utop opam ocamlPackages.dune m4 pkgconfig gmp hidapi
+ opam init --root=.
+ eval $(opam env)
  export PATH=_opam/bin:$PATH
  make build-dev-deps
  dune build src/bin_sandbox/main.exe
 ```
-I also usually tinker with the `test/run-flextesa-tests.sh` so that it gets the flextesa executable from `nix/dep/flextesa-dev/_build/default/src/bin_sandbox/main.exe` 
+I also usually tinker with the `test/run-flextesa-tests.sh` so that it gets the flextesa executable from `nix/dep/flextesa-dev/_build/default/src/bin_sandbox/main.exe`
```

Comment:
Do we need this? Also nix/dep is deleted.

Reply:
Probably, let's add this to the new opened issue.

Reply:
https://github.com/trilitech/ledger-app-tezos-baking/issues/55